config_explorer/src/config_explorer/capacity_planner.py

```diff
@@ -396,6 +396,30 @@ def kv_cache_req(model_info: ModelInfo,
 
     return KVCacheDetail(model_info, model_config, context_len, batch_size).kv_cache_size_gb
 
+def total_kv_cache_blocks(model_info: ModelInfo,
+                    model_config: AutoConfig,
+                    context_len: int,
+                    gpu_memory: int,
+                    gpu_mem_util: float=0.9,
+                    batch_size: int = 1,
+                    block_size: int = 16,
```

Comment:
I'm also wondering if we want to do input checking? vLLM only accepts certain values for block size. Future enhancement, perhaps, but would be good to make a note of it in docstring